cmd/verify.go

```diff
@@ -59,8 +59,10 @@ func runVerify(cmd *cobra.Command, args []string) (err error) {
 	values.Set("per_page", per_page)
 	// if provider was specified, filter results for just that provider. Otherwise, target all supported providers:
 	secret_type := getSecretTypeParameter()
-	values.Set("secret_type", secret_type)
-	parsedURL.RawQuery = values.Encode()
+	if secret_type != "all" {
+		values.Set("secret_type", secret_type)
+		parsedURL.RawQuery = values.Encode()
+	}
 
 	// update the request path
 	requestPath = parsedURL.String()
```

Comment:
I think we'll need to apply this same logic change to `alerts.go` lines 56:63

Reply:
I don't think that is necessary as the logic in `alerts.go` filters the results (alerts) to the secret types if selected, otherwise it already fetches and prints all alerts. Therefore the [current code](https://github.com/theztefan/gh-secret-scanning/blob/main/cmd/alerts.go#L55-L61) is correct imo.

Reply:
Ahh you're right, since the `provider` check is happening before calling `getSecretTypeParameter()`. Thanks for double checking! 🙇